specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-04-01-preview/examples/workspaceManagerAssignments/GetAllWorkspaceManagerAssignments.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "api-version": "2023-04-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/workspaceManagerAssignments/47cdc5f5-37c4-47b5-bd5f-83c84b8bdd58",
+            "etag": "\"190057d0-0000-0d00-0000-5c6f5adb0000\"",
+            "name": "47cdc5f5-37c4-47b5-bd5f-83c84b8bdd58",
+            "type": "Microsoft.SecurityInsights/workspaceManagerAssignments",
+            "properties": {
+              "items": [
+                {
+                  "resourceId": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspac-es/myWorkspace/providers/Microsoft.SecurityInsights/alertRules/microsoftSecurityIncidentCreationRuleExampleOne"
+                },
+                {
+                  "resourceId": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspac-es/myWorkspace/providers/Microsoft.SecurityInsights/alertRules/microsoftSecurityIncidentCreationRuleExampleTwo"
+                }
+              ],
+              "targetResourceName": "37207a7a-3b8a-438f-a559-c7df400e1b96",
```

Comment:
>targetResourceName [](http://example.com/codeflow?start=15&length=18)

I wasn't clear from the docs that it is going to be a guid? Is this because its an RBAC assignment id, so its an object id in AAD [or something]?

Reply:
This is the resource name of the workspace manager group targeted by the workspace manager assignment

Reply:
The name 'targetResource' is unfortunately pretty ambiguous as to what KIND of resource it means. How about calling it something along the lines of 'targetWorkspaceManagerGroupName' ?

Reply:
We made this targetResource due to the fact that in the future there is a possibility that we could point directly to a member & not just a group.